python/foglamp/services/core/api/utils.py

```diff
@@ -0,0 +1,30 @@
+import subprocess
+import os
+import json
+from foglamp.common import logger
+from foglamp.common.common import _FOGLAMP_ROOT
+
+_logger = logger.setup(__name__)
+
+
+def get_plugin_info(direction, name):
+    try:
+        # make
+        if os.path.isdir(_FOGLAMP_ROOT + '/cmake_build'):
```

Comment:
We shouldn't really do this, rather n=run the one under FOGLAMP_ROOT. There is nothing that says the developer has to build into cmake_build, that is just a convention in some people use.

Reply:
I raised this before in https://github.com/foglamp/FogLAMP/pull/975